libbeat/cfgfile/registry_test.go

```diff
@@ -1,48 +0,0 @@
-package cfgfile
```

Comment:
Any replacement for these tests?

Reply:
I forgot to add the file to the commit :man_facepalming: pushed